Release 1.1.12

Problems:
- [x] Nexus Staging Rules Failure: Missing SHA1
- [x] Github release update
- [ ] Push blocked

Release for `1.1.12` was [failed](http://www.rultor.com/t/16299-432170247) with
```
[ERROR] Rule failure while trying to close staging repository with ID "namevalery1707-1026".
[ERROR]
[ERROR] Nexus Staging Rules Failure Report
[ERROR] ==================================
[ERROR]
[ERROR] Repository "namevalery1707-1026" failures
[ERROR]   Rule "checksum-staging" failures
[ERROR]     * Missing SHA1: '/name/valery1707/jcommander-validator-number/0.1.12/jcommander-validator-number-0.1.12.pom.sha1'
```

And updating github release was failed to:
```
[INFO] Scanning for projects...
[INFO] Inspecting build with total of 1 modules...
[INFO] Not installing Nexus Staging features:
[INFO]  * Preexisting staging related goal bindings found in 1 modules.
[INFO] Downloading: https://repo.maven.apache.org/maven2/de/jutzig/github-release-plugin/1.2.0/github-release-plugin-1.2.0.pom
[INFO] Downloaded: https://repo.maven.apache.org/maven2/de/jutzig/github-release-plugin/1.2.0/github-release-plugin-1.2.0.pom (5 KB at 17.7 KB/sec)
[INFO] Downloading: https://repo.maven.apache.org/maven2/de/jutzig/github-release-plugin/1.2.0/github-release-plugin-1.2.0.jar
[INFO] Downloaded: https://repo.maven.apache.org/maven2/de/jutzig/github-release-plugin/1.2.0/github-release-plugin-1.2.0.jar (12 KB at 92.0 KB/sec)
[INFO]
[INFO] ------------------------------------------------------------------------
[INFO] Building JCommander extension library 0.2.0-SNAPSHOT
[INFO] ------------------------------------------------------------------------
[INFO]
[INFO] --- github-release-plugin:1.2.0:release (default-cli) @ jcommander-ext ---
[INFO] Downloading: https://repo.maven.apache.org/maven2/org/kohsuke/github-api/1.49/github-api-1.49.pom
[INFO] Downloaded: https://repo.maven.apache.org/maven2/org/kohsuke/github-api/1.49/github-api-1.49.pom (4 KB at 133.9 KB/sec)
[INFO] Downloading: https://repo.maven.apache.org/maven2/org/kohsuke/pom/6/pom-6.pom
[INFO] Downloaded: https://repo.maven.apache.org/maven2/org/kohsuke/pom/6/pom-6.pom (0 B at 0.0 KB/sec)
[INFO] Downloading: https://repo.maven.apache.org/maven2/commons-codec/commons-codec/1.7/commons-codec-1.7.pom
[INFO] Downloaded: https://repo.maven.apache.org/maven2/commons-codec/commons-codec/1.7/commons-codec-1.7.pom (11 KB at 360.2 KB/sec)
[INFO] Downloading: https://repo.maven.apache.org/maven2/org/apache/commons/commons-parent/26/commons-parent-26.pom
[INFO] Downloaded: https://repo.maven.apache.org/maven2/org/apache/commons/commons-parent/26/commons-parent-26.pom (47 KB at 1268.3 KB/sec)
[INFO] Downloading: https://repo.maven.apache.org/maven2/com/fasterxml/jackson/core/jackson-databind/2.2.3/jackson-databind-2.2.3.pom
[INFO] Downloaded: https://repo.maven.apache.org/maven2/com/fasterxml/jackson/core/jackson-databind/2.2.3/jackson-databind-2.2.3.pom (6 KB at 187.6 KB/sec)
[INFO] Downloading: https://repo.maven.apache.org/maven2/com/fasterxml/jackson/core/jackson-annotations/2.2.3/jackson-annotations-2.2.3.pom
[INFO] Downloaded: https://repo.maven.apache.org/maven2/com/fasterxml/jackson/core/jackson-annotations/2.2.3/jackson-annotations-2.2.3.pom (2 KB at 51.1 KB/sec)
[INFO] Downloading: https://repo.maven.apache.org/maven2/com/fasterxml/oss-parent/10/oss-parent-10.pom
[INFO] Downloaded: https://repo.maven.apache.org/maven2/com/fasterxml/oss-parent/10/oss-parent-10.pom (23 KB at 909.6 KB/sec)
[INFO] Downloading: https://repo.maven.apache.org/maven2/com/fasterxml/jackson/core/jackson-core/2.2.3/jackson-core-2.2.3.pom
[INFO] Downloaded: https://repo.maven.apache.org/maven2/com/fasterxml/jackson/core/jackson-core/2.2.3/jackson-core-2.2.3.pom (6 KB at 253.7 KB/sec)
[INFO] Downloading: https://repo.maven.apache.org/maven2/commons-io/commons-io/1.4/commons-io-1.4.pom
[INFO] Downloaded: https://repo.maven.apache.org/maven2/commons-io/commons-io/1.4/commons-io-1.4.pom (0 B at 0.0 KB/sec)
[INFO] Downloading: https://repo.maven.apache.org/maven2/org/apache/commons/commons-parent/7/commons-parent-7.pom
[INFO] Downloaded: https://repo.maven.apache.org/maven2/org/apache/commons/commons-parent/7/commons-parent-7.pom (0 B at 0.0 KB/sec)
[INFO] Downloading: https://repo.maven.apache.org/maven2/org/kohsuke/github-api/1.49/github-api-1.49.jar
[INFO] Downloading: https://repo.maven.apache.org/maven2/commons-codec/commons-codec/1.7/commons-codec-1.7.jar
[INFO] Downloading: https://repo.maven.apache.org/maven2/com/fasterxml/jackson/core/jackson-databind/2.2.3/jackson-databind-2.2.3.jar
[INFO] Downloading: https://repo.maven.apache.org/maven2/com/fasterxml/jackson/core/jackson-annotations/2.2.3/jackson-annotations-2.2.3.jar
[INFO] Downloading: https://repo.maven.apache.org/maven2/com/fasterxml/jackson/core/jackson-core/2.2.3/jackson-core-2.2.3.jar
[INFO] Downloaded: https://repo.maven.apache.org/maven2/org/kohsuke/github-api/1.49/github-api-1.49.jar (118 KB at 2504.7 KB/sec)
[INFO] Downloading: https://repo.maven.apache.org/maven2/commons-io/commons-io/1.4/commons-io-1.4.jar
[INFO] Downloaded: https://repo.maven.apache.org/maven2/commons-io/commons-io/1.4/commons-io-1.4.jar (0 B at 0.0 KB/sec)
[INFO] Downloading: https://repo.maven.apache.org/maven2/org/codehaus/plexus/plexus-utils/1.1/plexus-utils-1.1.jar
[INFO] Downloaded: https://repo.maven.apache.org/maven2/com/fasterxml/jackson/core/jackson-annotations/2.2.3/jackson-annotations-2.2.3.jar (33 KB at 503.0 KB/sec)
[INFO] Downloaded: https://repo.maven.apache.org/maven2/org/codehaus/plexus/plexus-utils/1.1/plexus-utils-1.1.jar (0 B at 0.0 KB/sec)
[INFO] Downloaded: https://repo.maven.apache.org/maven2/com/fasterxml/jackson/core/jackson-core/2.2.3/jackson-core-2.2.3.jar (189 KB at 1980.9 KB/sec)
[INFO] Downloaded: https://repo.maven.apache.org/maven2/commons-codec/commons-codec/1.7/commons-codec-1.7.jar (254 KB at 2130.4 KB/sec)
[INFO] Downloaded: https://repo.maven.apache.org/maven2/com/fasterxml/jackson/core/jackson-databind/2.2.3/jackson-databind-2.2.3.jar (846 KB at 3758.0 KB/sec)
[INFO] Creating release v0.1.12
[INFO] Processing asset /home/r/repo/validator/target/jcommander-validator-0.1.12.pom.asc
[INFO]   Upload asset
[INFO] Processing asset /home/r/repo/validator/target/jcommander-validator-0.1.12.jar
[INFO]   Upload asset
[INFO] Processing asset /home/r/repo/validator/target/jcommander-validator-0.1.12.jar.asc
[INFO]   Upload asset
[INFO] Processing asset /home/r/repo/validator/target/jcommander-validator-0.1.12.pom
[INFO]   Upload asset
[INFO] Processing asset /home/r/repo/validator-nio/target/jcommander-validator-nio-0.1.12-sources.jar.asc
[INFO]   Upload asset
[INFO] Processing asset /home/r/repo/validator-nio/target/jcommander-validator-nio-0.1.12.pom.asc
[INFO]   Upload asset
[INFO] Processing asset /home/r/repo/validator-nio/target/jcommander-validator-nio-0.1.12.jar.asc
[INFO]   Upload asset
[INFO] Processing asset /home/r/repo/validator-nio/target/jcommander-validator-nio-0.1.12.jar
[INFO]   Upload asset
[INFO] Processing asset /home/r/repo/validator-nio/target/jcommander-validator-nio-0.1.12.pom
[INFO]   Upload asset
[INFO] Processing asset /home/r/repo/validator-nio/target/jcommander-validator-nio-0.1.12-javadoc.jar
[INFO]   Upload asset
[INFO] Processing asset /home/r/repo/validator-nio/target/jcommander-validator-nio-0.1.12-sources.jar
[INFO]   Upload asset
[INFO] Processing asset /home/r/repo/validator-nio/target/jcommander-validator-nio-0.1.12-javadoc.jar.asc
[INFO]   Upload asset
[INFO] Processing asset /home/r/repo/validator-string/target/jcommander-validator-string-0.1.12.pom
[INFO]   Upload asset
[INFO] Processing asset /home/r/repo/validator-string/target/jcommander-validator-string-0.1.12.jar
[INFO]   Upload asset
[INFO] Processing asset /home/r/repo/validator-string/target/jcommander-validator-string-0.1.12-sources.jar
[INFO]   Upload asset
[INFO] Processing asset /home/r/repo/validator-string/target/jcommander-validator-string-0.1.12.jar.asc
[INFO]   Upload asset
[INFO] Processing asset /home/r/repo/validator-string/target/jcommander-validator-string-0.1.12-javadoc.jar
[INFO]   Upload asset
[INFO] Processing asset /home/r/repo/validator-string/target/jcommander-validator-string-0.1.12-sources.jar.asc
[INFO]   Upload asset
[INFO] Processing asset /home/r/repo/validator-string/target/jcommander-validator-string-0.1.12-javadoc.jar.asc
[INFO]   Upload asset
[INFO] Processing asset /home/r/repo/validator-string/target/jcommander-validator-string-0.1.12.pom.asc
[INFO]   Upload asset
[INFO] Processing asset /home/r/repo/target/jcommander-ext-0.1.12.pom.asc
[INFO]   Upload asset
[INFO] Processing asset /home/r/repo/target/jcommander-ext-0.1.12.pom
[INFO]   Upload asset
[INFO] Processing asset /home/r/repo/target/nexus-staging/staging/5bd39d184132d/name/valery1707/jcommander-validator-core/0.1.12/jcommander-validator-core-0.1.12-sources.jar
[INFO]   Upload asset
[INFO] Processing asset /home/r/repo/target/nexus-staging/staging/5bd39d184132d/name/valery1707/jcommander-validator-core/0.1.12/jcommander-validator-core-0.1.12-javadoc.jar
[INFO]   Upload asset
[INFO] Processing asset /home/r/repo/target/nexus-staging/staging/5bd39d184132d/name/valery1707/jcommander-validator-core/0.1.12/jcommander-validator-core-0.1.12.jar
[INFO]   Upload asset
[INFO] Processing asset /home/r/repo/target/nexus-staging/staging/5bd39d184132d/name/valery1707/jcommander-validator-core/0.1.12/jcommander-validator-core-0.1.12.pom
[INFO]   Upload asset
[INFO] Processing asset /home/r/repo/target/nexus-staging/staging/5bd39d184132d/name/valery1707/jcommander-validator-core/0.1.12/jcommander-validator-core-0.1.12-sources.jar.asc
[INFO]   Upload asset
[INFO] Processing asset /home/r/repo/target/nexus-staging/staging/5bd39d184132d/name/valery1707/jcommander-validator-core/0.1.12/jcommander-validator-core-0.1.12-javadoc.jar.asc
[INFO]   Upload asset
[INFO] Processing asset /home/r/repo/target/nexus-staging/staging/5bd39d184132d/name/valery1707/jcommander-validator-core/0.1.12/jcommander-validator-core-0.1.12.jar.asc
[INFO]   Upload asset
[INFO] Processing asset /home/r/repo/target/nexus-staging/staging/5bd39d184132d/name/valery1707/jcommander-validator-core/0.1.12/jcommander-validator-core-0.1.12.pom.asc
[INFO]   Upload asset
[INFO] Processing asset /home/r/repo/target/nexus-staging/staging/5bd39d184132d/name/valery1707/jcommander-validator-io/0.1.12/jcommander-validator-io-0.1.12-sources.jar.asc
[INFO]   Upload asset
[INFO] Processing asset /home/r/repo/target/nexus-staging/staging/5bd39d184132d/name/valery1707/jcommander-validator-io/0.1.12/jcommander-validator-io-0.1.12-javadoc.jar
[INFO]   Upload asset
[INFO] Processing asset /home/r/repo/target/nexus-staging/staging/5bd39d184132d/name/valery1707/jcommander-validator-io/0.1.12/jcommander-validator-io-0.1.12.jar.asc
[INFO]   Upload asset
[INFO] Processing asset /home/r/repo/target/nexus-staging/staging/5bd39d184132d/name/valery1707/jcommander-validator-io/0.1.12/jcommander-validator-io-0.1.12-javadoc.jar.asc
[INFO]   Upload asset
[INFO] Processing asset /home/r/repo/target/nexus-staging/staging/5bd39d184132d/name/valery1707/jcommander-validator-io/0.1.12/jcommander-validator-io-0.1.12-sources.jar
[INFO]   Upload asset
[INFO] Processing asset /home/r/repo/target/nexus-staging/staging/5bd39d184132d/name/valery1707/jcommander-validator-io/0.1.12/jcommander-validator-io-0.1.12.pom.asc
[INFO]   Upload asset
[INFO] Processing asset /home/r/repo/target/nexus-staging/staging/5bd39d184132d/name/valery1707/jcommander-validator-io/0.1.12/jcommander-validator-io-0.1.12.pom
[INFO]   Upload asset
[INFO] Processing asset /home/r/repo/target/nexus-staging/staging/5bd39d184132d/name/valery1707/jcommander-validator-io/0.1.12/jcommander-validator-io-0.1.12.jar
[INFO]   Upload asset
[INFO] Processing asset /home/r/repo/target/nexus-staging/staging/5bd39d184132d/name/valery1707/jcommander-ext/0.1.12/jcommander-ext-0.1.12.pom.asc
[WARNING] Asset jcommander-ext-0.1.12.pom.asc already exists. Skipping
[INFO] Processing asset /home/r/repo/target/nexus-staging/staging/5bd39d184132d/name/valery1707/jcommander-ext/0.1.12/jcommander-ext-0.1.12.pom
[WARNING] Asset jcommander-ext-0.1.12.pom already exists. Skipping
[INFO] Processing asset /home/r/repo/target/nexus-staging/staging/5bd39d184132d/name/valery1707/jcommander-validator-nio/0.1.12/jcommander-validator-nio-0.1.12-sources.jar.asc
[WARNING] Asset jcommander-validator-nio-0.1.12-sources.jar.asc already exists. Skipping
[INFO] Processing asset /home/r/repo/target/nexus-staging/staging/5bd39d184132d/name/valery1707/jcommander-validator-nio/0.1.12/jcommander-validator-nio-0.1.12.pom.asc
[WARNING] Asset jcommander-validator-nio-0.1.12.pom.asc already exists. Skipping
[INFO] Processing asset /home/r/repo/target/nexus-staging/staging/5bd39d184132d/name/valery1707/jcommander-validator-nio/0.1.12/jcommander-validator-nio-0.1.12.jar.asc
[WARNING] Asset jcommander-validator-nio-0.1.12.jar.asc already exists. Skipping
[INFO] Processing asset /home/r/repo/target/nexus-staging/staging/5bd39d184132d/name/valery1707/jcommander-validator-nio/0.1.12/jcommander-validator-nio-0.1.12.jar
[WARNING] Asset jcommander-validator-nio-0.1.12.jar already exists. Skipping
[INFO] Processing asset /home/r/repo/target/nexus-staging/staging/5bd39d184132d/name/valery1707/jcommander-validator-nio/0.1.12/jcommander-validator-nio-0.1.12.pom
[WARNING] Asset jcommander-validator-nio-0.1.12.pom already exists. Skipping
[INFO] Processing asset /home/r/repo/target/nexus-staging/staging/5bd39d184132d/name/valery1707/jcommander-validator-nio/0.1.12/jcommander-validator-nio-0.1.12-javadoc.jar
[WARNING] Asset jcommander-validator-nio-0.1.12-javadoc.jar already exists. Skipping
[INFO] Processing asset /home/r/repo/target/nexus-staging/staging/5bd39d184132d/name/valery1707/jcommander-validator-nio/0.1.12/jcommander-validator-nio-0.1.12-sources.jar
[WARNING] Asset jcommander-validator-nio-0.1.12-sources.jar already exists. Skipping
[INFO] Processing asset /home/r/repo/target/nexus-staging/staging/5bd39d184132d/name/valery1707/jcommander-validator-nio/0.1.12/jcommander-validator-nio-0.1.12-javadoc.jar.asc
[WARNING] Asset jcommander-validator-nio-0.1.12-javadoc.jar.asc already exists. Skipping
[INFO] Processing asset /home/r/repo/target/nexus-staging/staging/5bd39d184132d/name/valery1707/jcommander-validator/0.1.12/jcommander-validator-0.1.12.pom.asc
[WARNING] Asset jcommander-validator-0.1.12.pom.asc already exists. Skipping
[INFO] Processing asset /home/r/repo/target/nexus-staging/staging/5bd39d184132d/name/valery1707/jcommander-validator/0.1.12/jcommander-validator-0.1.12.jar
[WARNING] Asset jcommander-validator-0.1.12.jar already exists. Skipping
[INFO] Processing asset /home/r/repo/target/nexus-staging/staging/5bd39d184132d/name/valery1707/jcommander-validator/0.1.12/jcommander-validator-0.1.12.jar.asc
[WARNING] Asset jcommander-validator-0.1.12.jar.asc already exists. Skipping
[INFO] Processing asset /home/r/repo/target/nexus-staging/staging/5bd39d184132d/name/valery1707/jcommander-validator/0.1.12/jcommander-validator-0.1.12.pom
[WARNING] Asset jcommander-validator-0.1.12.pom already exists. Skipping
[INFO] Processing asset /home/r/repo/target/nexus-staging/staging/5bd39d184132d/name/valery1707/jcommander-validator-string/0.1.12/jcommander-validator-string-0.1.12.pom
[INFO]   Upload asset
[ERROR]
java.io.IOException: {"message":"Validation Failed","request_id":"9F46:68DD:9351BA:A4C985:5BCEEA9D","documentation_url":"https://developer.github.com/v3","errors":[{"resource":"ReleaseAsset","code&quot;:&quot;already_exists&quot;,&quot;field&quot;:&quot;name&quot;}]}
        at org.kohsuke.github.Requester.handleApiError(Requester.java:367)
        at org.kohsuke.github.Requester._to(Requester.java:202)
        at org.kohsuke.github.Requester.to(Requester.java:154)
        at org.kohsuke.github.GHRelease.uploadAsset(GHRelease.java:182)
        at de.jutzig.github.release.plugin.UploadMojo.uploadAsset(UploadMojo.java:236)
        at de.jutzig.github.release.plugin.UploadMojo.uploadAssets(UploadMojo.java:246)
        at de.jutzig.github.release.plugin.UploadMojo.execute(UploadMojo.java:205)
        at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:134)
        at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:207)
        at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:153)
        at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:145)
        at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:116)
        at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:80)
        at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:51)
        at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:128)
        at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:307)
        at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:193)
        at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:106)
        at org.apache.maven.cli.MavenCli.execute(MavenCli.java:863)
        at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:288)
        at org.apache.maven.cli.MavenCli.main(MavenCli.java:199)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:289)
        at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:229)
        at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:415)
        at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:356)
Caused by: java.io.IOException: Server returned HTTP response code: 422 for URL: https://uploads.github.com/repos/valery1707/jcommander-ext/releases/13601613/assets?name=jcommander-validator-string-0.1.12.pom
        at sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1894)
        at sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1492)
        at sun.net.www.protocol.https.HttpsURLConnectionImpl.getInputStream(HttpsURLConnectionImpl.java:263)
        at org.kohsuke.github.Requester.parse(Requester.java:323)
        at org.kohsuke.github.Requester._to(Requester.java:200)
        ... 27 more
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 35.932 s
[INFO] Finished at: 2018-10-23T09:32:16+00:00
[INFO] Final Memory: 20M/360M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal de.jutzig:github-release-plugin:1.2.0:release (default-cli) on project jcommander-ext: Failed to upload assets: {"message":"Validation Failed","request_id":"9F46:68DD:9351BA:A4C985:5BCEEA9D","documentation_url":"https://developer.github.com/v3","errors":[{"resource":"ReleaseAsset","code":"already_exists","field":"name"}]}: Server returned HTTP response code: 422 for URL: https://uploads.github.com/repos/valery1707/jcommander-ext/releases/13601613/assets?name=jcommander-validator-string-0.1.12.pom -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
```

--- a/.rultor/deploy.sh
+++ b/.rultor/deploy.sh
@@ -1,4 +1,5 @@
 #!/usr/bin/env bash
+set -e
 
 version=${tag}
 # Release version

--- a/pom.xml
+++ b/pom.xml
@@ -159,7 +159,43 @@
 					<generateBackupPoms>false</generateBackupPoms>
 				</configuration>
 			</plugin>
+
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-install-plugin</artifactId>
+				<version>2.5.2</version>
+				<configuration>
+					<!--Still not worked for unknown reason. Use separate plugin for checksum generation.-->
+					<createChecksum>true</createChecksum>
+				</configuration>
+			</plugin>
+
+			<plugin>
+				<groupId>net.nicoulaj.maven.plugins</groupId>
+				<artifactId>checksum-maven-plugin</artifactId>
+			</plugin>
 		</plugins>
+
+		<pluginManagement>
+			<plugins>
+				<plugin>
+					<groupId>net.nicoulaj.maven.plugins</groupId>
+					<artifactId>checksum-maven-plugin</artifactId>
+					<version>1.8</version>
+					<executions>
+						<execution>
+							<goals>
+								<goal>artifacts</goal>
+							</goals>
+						</execution>
+					</executions>
+					<configuration>
+						<!--Root project don't have any artifact-->
+						<failOnError>false</failOnError>
+					</configuration>
+				</plugin>
+			</plugins>
+		</pluginManagement>
 	</build>
 
 	<distributionManagement>
@@ -195,7 +231,7 @@
 					<plugin>
 						<groupId>org.apache.maven.plugins</groupId>
 						<artifactId>maven-javadoc-plugin</artifactId>
-						<version>2.10.4</version>
+						<version>3.0.1</version>
 						<executions>
 							<execution>
 								<id>attach-javadocs</id>

--- a/pom.xml
+++ b/pom.xml
@@ -305,6 +305,7 @@
 									</includes>
 									<excludes>
 										<exclude>**/jcommander-*-test-*.*</exclude>
+										<exclude>**/staging/**</exclude>
 									</excludes>
 								</fileSet>
 							</fileSets>

--- a/validator-core/src/test/java/name/valery1707/jcommander/validators/ParameterExceptionsTest.java
+++ b/validator-core/src/test/java/name/valery1707/jcommander/validators/ParameterExceptionsTest.java
@@ -7,12 +7,12 @@ import static org.assertj.core.api.Assertions.assertThat;
 
 public class ParameterExceptionsTest {
 	@Test(expected = UnsupportedOperationException.class)
-	public void createDirectly() throws Exception {
+	public void createDirectly() {
 		new ParameterExceptions();
 	}
 
 	@Test
-	public void shouldThrow() throws Exception {
+	public void shouldThrow() {
 		assertThat(ParameterExceptions.invalidParameter("name", "desc"))
 				.isInstanceOf(ParameterException.class)
 				.hasMessageContaining("'name'")

--- a/validator-core/src/test/java/name/valery1707/jcommander/validators/common/InRangeFormatTest.java
+++ b/validator-core/src/test/java/name/valery1707/jcommander/validators/common/InRangeFormatTest.java
@@ -7,44 +7,44 @@ import static org.assertj.core.api.Assertions.assertThat;
 
 public class InRangeFormatTest {
 	@Test
-	public void rangeClosedClosed() throws Exception {
+	public void rangeClosedClosed() {
 		assertThat(new InRangeSimple(1, true, 5, true).error()).endsWith("range [1, 5]");
 	}
 
 	@Test
-	public void rangeClosedOpen() throws Exception {
+	public void rangeClosedOpen() {
 		assertThat(new InRangeSimple(1, true, 5, false).error()).endsWith("range [1, 5)");
 	}
 
 	@Test
-	public void rangeOpenClosed() throws Exception {
+	public void rangeOpenClosed() {
 		assertThat(new InRangeSimple(1, false, 5, true).error()).endsWith("range (1, 5]");
 	}
 
 	@Test
-	public void rangeOpenOpen() throws Exception {
+	public void rangeOpenOpen() {
 		assertThat(new InRangeSimple(1, false, 5, false).error()).endsWith("range (1, 5)");
 	}
 
 	@Test
-	public void rangeFromInfinity() throws Exception {
+	public void rangeFromInfinity() {
 		assertThat(new InRangeSimple(null, false, 5, false).error()).endsWith("range (..., 5)");
 		assertThat(new InRangeSimple(null, true, 5, true).error()).endsWith("range (..., 5]");
 	}
 
 	@Test
-	public void rangeToInfinity() throws Exception {
+	public void rangeToInfinity() {
 		assertThat(new InRangeSimple(1, false, null, false).error()).endsWith("range (1, ...)");
 		assertThat(new InRangeSimple(1, true, null, true).error()).endsWith("range [1, ...)");
 	}
 
 	@Test(expected = IllegalArgumentException.class)
-	public void rangeReversed() throws Exception {
+	public void rangeReversed() {
 		new InRangeSimple(5, true, 1, true);
 	}
 
 	@Test(expected = IllegalArgumentException.class)
-	public void rangeUnBordered() throws Exception {
+	public void rangeUnBordered() {
 		new InRangeSimple(null, true, null, true);
 	}
 }

--- a/validator-nio/src/test/java/name/valery1707/jcommander/validators/nio/VirtualFolder.java
+++ b/validator-nio/src/test/java/name/valery1707/jcommander/validators/nio/VirtualFolder.java
@@ -55,7 +55,7 @@ public class VirtualFolder extends ExternalResource {
 	}
 
 	public enum FileType {
-		FILE, DIRECTORY, LINK;
+		FILE, DIRECTORY, LINK
 	}
 
 	@SuppressWarnings("WeakerAccess")

--- a/validator-test/src/main/java/name/valery1707/jcommander/validators/ValidatorTest.java
+++ b/validator-test/src/main/java/name/valery1707/jcommander/validators/ValidatorTest.java
@@ -21,7 +21,7 @@ public abstract class ValidatorTest<T, V extends IValueValidator<T>> {
 	}
 
 	@Before
-	public void setUp() throws Exception {
+	public void setUp() {
 		validator = createValidator();
 	}
 
@@ -31,7 +31,7 @@ public abstract class ValidatorTest<T, V extends IValueValidator<T>> {
 	private final boolean isValid;
 
 	@Test
-	public void checkData() throws Exception {
+	public void checkData() {
 		Description description = new TextDescription("Validator must %s value '%s'", (isValid ? "accept" : "decline"), String.valueOf(value));
 		try {
 			validator.validate("-check", value);

